### ISSUE TITLE
I would like known why v8ClassificationLoss need to divide a fixed value 64

### ISSUE THREAD

**Issue by CavaliersFor** [ultralytics/ultralytics#3636]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi Ultralytics team! 
I would like known why v8ClassificationLoss need to divide a fixed value 64?

I think this should be divided by the corresponding batch size, rather than a fixed value. 

```
class v8ClassificationLoss:

    def __call__(self, preds, batch):
        """Compute the classification loss between predictions and true labels."""
        loss = torch.nn.functional.cross_entropy(preds, batch['cls'], reduction='sum') / 64
        loss_items = loss.detach()
        return loss, loss_items
```
loss = torch.nn.functional.cross_entropy(preds, batch['cls'], reduction='sum') /  **64**

**Comment by glenn-jocher** [comment]
@CavaliersFor hi there! Thank you for your question about the `v8ClassificationLoss` in YOLOv8. The division by a fixed value of 64 is not specifically tied to the batch size, but is instead used to normalize the loss value.

The division by 64 is a scaling factor that helps to balance the magnitude of the loss with respect to other components of the YOLOv8 loss function. It is a common practice to normalize the loss to avoid numerical instability and ensure that all components of the loss contribute meaningfully to the overall optimization process.

While dividing by the batch size can also help achieve a similar effect, using a fixed value like 64 allows for more consistent and controlled behavior across different batch sizes. It is worth mentioning that the choice of the specific value, in this case 64, may be empirical and depends on the problem being solved and the specific network architecture.

I hope this explanation clarifies the purpose of dividing by 64 in the `v8ClassificationLoss` function. Let us know if you have any further questions or if there's anything else we can assist you with!
